src/support/components/PayGSupportOverlay.tsx

```diff
@@ -96,20 +100,34 @@ const PayGSupportOverlay: FC<OwnProps> = () => {
   }
 
   const handleSubmit = async () => {
-    const email = quartzMe?.email
-    const descriptionWithOrgId = `${description}  [Org Id: ${orgID}]`
+    let userID: string, userEmail: string, orgName: string, orgID: string
+
+    if (shouldUseQuartzIdentity()) {
+      userID = identityUser.id
+      userEmail = identityUser.email
+      orgName = identityOrg.name
+      orgID = identityOrg.id
+    } else {
+      // Optional chaining operator needed because quartzMe's initial reducer state is null.
+      userID = quartzMe?.id
+      userEmail = quartzMe?.email
+      orgName = quartzOrg?.name
+      orgID = quartzOrg?.id
+    }
+
+    const descriptionWithOrgId = `[Org Name: ${orgName}] [Org Id: ${orgID}] \n\n ${description}`
     const translatedSeverity = translateSeverityLevelForSfdc(severity)
     try {
       await createSfdcSupportCase(
         descriptionWithOrgId,
-        email,
+        userEmail,
         translatedSeverity,
         subject
       )
       event(
         'helpBar.paygSupportRequest.submitted',
         {},
-        {userID: me.id, orgID: orgID}
+        {userID: userID, orgName: orgName, orgID: orgID}
```

Comment:
minor style nit, but just use that good ole es6 object property shorthand

```suggestion
        {userID, orgName, orgID}
```